app/controllers/test/components/mentoring/mentor_solutions_list/solutions_controller.rb

```diff
@@ -0,0 +1,70 @@
+class Test::Components::Mentoring::MentorSolutionsList::SolutionsController < ApplicationController
```

Comment:
As per my comment on the other PR, please change this to be a `solutions` action on `Test::Components::Mentoring::MentorSolutionsList`:)